app/models/draft.rb

```diff
@@ -43,6 +43,43 @@ def self.find_draft(user, key)
     end
   end
 
+  def self.stream(opts = nil)
+    opts ||= {}
+
+    user_id = opts[:user_id]
+    offset = opts[:offset] || 0
+    limit = opts[:limit] || 30
```

Comment:
It looks like the default chunk size is duplicated here and in the controller. I think we should only have one location for this logic.